roles/java/tasks/keystores.yml

```diff
@@ -28,6 +28,11 @@
         mode: "0400"
         dest: "{{ download_location }}"
       register: certcp
+    - name: Assign variable based on distribution
+      ansible.builtin.set_fact:
+        legacy: "-legacy"
```

Comment:
call the variable something like `additional_options` so it's more explicit what we are doing